src/filters/out_route.c

```diff
@@ -2117,11 +2170,16 @@ void routeout_send_fdt(GF_ROUTEOutCtx *ctx, ROUTEService *serv, ROUTEPid *rpid)
 	gf_dynstrcat(&payload, "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"no\" ?>\n", NULL);
 	gf_dynstrcat(&payload, "<FDT-Instance Expires=\"3916741152\" xmlns=\"urn:IETF:metadata:2005:FLUTE:FDT\">\n", NULL);
 	//todo: inject manifest, init seg and child playlist ?
```

Comment:
can you remove the todo comment now that this is done ?

Reply:
Done